### PR TITLE
Document the cratedb docker healthcheck.

### PR DESCRIPTION
**Pull request by andreidan**

### CODE REVIEW

File: docs/deployment/containers/docker.txt [hunk 1]
```diff
@@ -191,6 +191,13 @@ Here's how you would mount ``crate.yml``::
 Here, ``/srv/crate/config/crate.yml`` is an example path, and should be
 replaced with the path to your host machine ``crate.yml`` file.
 
+Healthcheck
+-----------
+
+The official `CrateDB Docker image`_ ships with a configured liveness
+`Docker healthcheck`_. The purpose of this check is to signal the situations
+when the CrateDB process might've crashed or hanged inside the container
```
Comment (matriv) [inline docs/deployment/containers/docker.txt]: `'ve` to `have`

File: docs/deployment/containers/docker.txt [hunk 2]
```diff
@@ -191,6 +191,13 @@ Here's how you would mount ``crate.yml``::
 Here, ``/srv/crate/config/crate.yml`` is an example path, and should be
 replaced with the path to your host machine ``crate.yml`` file.
 
+Healthcheck
+-----------
+
+The official `CrateDB Docker image`_ ships with a configured liveness
+`Docker healthcheck`_. The purpose of this check is to signal the situations
```
Comment (matriv) [inline docs/deployment/containers/docker.txt]: `signal` -> `indicate`

Comment (andreidan) [reply docs/deployment/containers/docker.txt]: @nslater what do you think? I find `signal` to be expressive enough

File: docs/deployment/containers/docker.txt [hunk 3]
```diff
@@ -191,6 +191,13 @@ Here's how you would mount ``crate.yml``::
 Here, ``/srv/crate/config/crate.yml`` is an example path, and should be
 replaced with the path to your host machine ``crate.yml`` file.
 
+Healthcheck
+-----------
+
+The official `CrateDB Docker image`_ ships with a configured liveness
```
Comment (nslater) [inline docs/deployment/containers/docker.txt]: drop "liveness"

File: docs/deployment/containers/docker.txt (hunk 2 above)
Comment (nslater) [reply docs/deployment/containers/docker.txt]: This health check will flag that CrateDB process might...

Comment (nslater) [reply docs/deployment/containers/docker.txt]: also change it to "Docker health check"

File: docs/deployment/containers/docker.txt (hunk 3 above)
Comment (andreidan) [reply docs/deployment/containers/docker.txt]: I don't agree with this as there can be multiple types of checks (ie. readiness - which would indicate if crate is queryable). I think it's appropriate to specify what kind of check this is.

Comment (nslater) [reply docs/deployment/containers/docker.txt]: is "liveness" a standard term? I would have expected it to be "liveliness"

in any case, I am not sure it makes sense to disambiguate here unless we have multiple health checks to offer or that there's a reasonable chance of confusion

surely someone who's confused or doesn't know would follow the docs and learn what *we* mean by health checks?

also doesn't this allow us to expand the scope of our health checks in future?

Comment (nslater) [reply docs/deployment/containers/docker.txt]: just been linked to kube docs using "liveness" in which case okay

keep this :)

**Pull request merged**